seguridad/views.py

```
@@ -25,6 +25,26 @@ def post(self, request, *args, **kwargs):
+        data = {'resp': False}
+        try:
+            cuenta = str(request.POST.get('usuario')).strip()
+            user = authenticate()
+            if user is not None:
+                if user.is_active:
+                    login(request, user)
+                    data['resp'] = True
+                    data['user'] = user.username
+                    return JsonResponse(data, status=200)
+                else:
+                    data['error'] = 'Login Fallido!, usuario no esta habilitado'
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)